plugins/hydra_ray_launcher/examples/docker/config.yaml

```diff
@@ -0,0 +1,48 @@
+task: 1
+
+hydra:
+  launcher:
+    _target_: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
+    env_setup:
+      pip_packages:
+        omegaconf: null
+        hydra_core: null
+        ray: null
+        cloudpickle: null
+        pickle5: null
+        hydra_ray_launcher: null
+      commands: []
+    ray:
+      cluster:
+        cluster_name: example-cluster
+        min_workers: 0
+        max_workers: 0
+        initial_workers: 0
+        autoscaling_mode: default
+        target_utilization_fraction: 0.8
+        idle_timeout_minutes: 5
+        docker:
+          image: 'samuelstanton/ray-plugin-example'
+          container_name: 'hydra-container'
+          pull_before_run: true
+          run_options: []
+        provider:
+          type: aws
+          region: us-east-2
+          availability_zone: us-east-2a,us-east-2b
+          cache_stopped_nodes: false
+          key_pair:
+            key_name: hydra
```

Comment:
These are in default config, I don't think we need to add them here.

Reply:
There's a balance between keeping the example simple by not including redundant elements of the config, and making the example easy for the user to adapt. It is very likely that some users will not want to use the `us-east-2` region, and they shouldn't have to dig through the source to figure out how to change it.